dev/benchmarks/macrobenchmarks/web/flutter_bootstrap.js

```diff
@@ -1,8 +1,13 @@
 {{flutter_js}}
 {{flutter_build_config}}
+
+const searchParams = new URLSearchParams(window.location.search);
+const forceSt = searchParams.get('force_st');
+const extraConfig = forceSt ? {forceSingleThreadedSkwasm: true} : {};
 _flutter.loader.load({
   config: {
     // Use the local CanvasKit bundle instead of the CDN to reduce test flakiness.
     canvasKitBaseUrl: "/canvaskit/",
+    ...extraConfig,
```

Comment:
Maybe:

```js
forceSingleThreadedSkwasm: !!forceSt
```

(Is there any difference between not passing the `forceSingleThreadedSkwasm` or making it `false`?)

Reply:
I don't think it makes much of a difference either way, but I was hoping not to modify the config at all for existing benchmarks.